index.js

```diff
@@ -55,14 +55,33 @@ module.exports = postcss.plugin('postcss-increase-specificity', function(options
 	var opts = objectAssign({}, defaults, options);
```

Comment:
Sorry for the delay @mihaisavezi 🙇 

This looks like a great v1 we can merge of this feature 👍

My comments are mainly framing everything in terms of `disable`/`enable` instead of `exclude`